conveyor/src/main/kotlin/com/hodamohammadi/conveyor/activities/AuthenticationActivity.kt

```diff
@@ -1,98 +1,48 @@
 package com.hodamohammadi.conveyor.activities
 
 import android.support.v7.app.AppCompatActivity
-import com.google.android.gms.common.api.GoogleApiClient
-import android.widget.Toast
-import com.google.android.gms.common.ConnectionResult
 import android.content.Intent
-import com.google.firebase.auth.GoogleAuthProvider
-import com.google.android.gms.auth.api.signin.GoogleSignInAccount
-import com.google.android.gms.auth.api.Auth
-import com.google.firebase.auth.FirebaseAuth
-import com.google.android.gms.auth.api.signin.GoogleSignInOptions
 import android.os.Bundle
-import android.util.Log
+import android.widget.Toast
 import com.hodamohammadi.conveyor.R
-import kotlinx.android.synthetic.main.authentication_activity.*
+import com.firebase.ui.auth.AuthUI
+import com.firebase.ui.auth.ErrorCodes
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/itsAlexNguyen/Conveyor/blob/847879133c27dfb61c2a596fe4545276e3aa595a/conveyor/src/main/kotlin/com/hodamohammadi/conveyor/activities/AuthenticationActivity.kt#L9"></span>The import ‘com.firebase.ui.auth.ErrorCodes’ is unused.</td>
    </tr>
  </tbody>
</table>